jboss/container/wildfly/standalone-conf/java/added/standalone.conf

```diff
@@ -1,15 +0,0 @@
-
-source /usr/local/dynamic-resources/dynamic_resources.sh
-export GC_MAX_METASPACE_SIZE=${GC_MAX_METASPACE_SIZE:-256}
-JAVA_OPTS="$(adjust_java_options ${JAVA_OPTS})"
-
-# Make sure that we use /dev/urandom (CLOUD-422)
-JAVA_OPTS="${JAVA_OPTS} -Djava.security.egd=file:/dev/./urandom"
```

Comment:
Did -Djava.security.egd=file:/dev/./urandom get moved somewhere else, or is it not needed anymore?

Reply:
yes, all was moved some time ago....https://github.com/wildfly/wildfly-cekit-modules/blob/master/jboss/container/wildfly/galleon/fp-content/java/added/src/main/resources/packages/wildfly.s2i.java/content/bin/java-conf.conf

Reply:
Got it, thanks!